js/app/improve-this-page.js

```diff
@@ -0,0 +1,67 @@
+$(document).ready(function() {
+    var pageURL = window.location.href;
+
+    $( "#feedback-form-url" ).val(pageURL);
+
+    $( "#feedback-form-no" ).click(function(e) {
+        e.preventDefault();
+
+        $( "#feedback-form" ).removeClass("js-hidden");
+        $( "#feedback-form-header" ).addClass("js-hidden");
+        $(" #description-field ").focus();
+    });
+
+    $( "#feedback-form-anything-wrong" ).click(function(e) {
```

Comment:
The three click events could me reduced by adding some conditional statements, possibly toggleClass on the js-hidden class...would have to try it out. 

I would create a common class to use across the three elements and then check for which one has been clicked  by it's ID.